src/components/pages/HeadsUpDisplay/CurrentActivity.js

```diff
@@ -1,117 +1,101 @@
+        setDay(10);
+        return;
+      }
+    } else if (today === 4) {
+      if (day === 3) {
+        setDay(4);
+        return;
+      } else if (day === 10) {
+        setDay(11);
+        return;
+      }
+    } else if (today === 5) {
+      if (day === 4) {
+        setDay(5);
+        return;
+      } else if (day === 11) {
+        setDay(12);
+        return;
+      }
+    } else if (today === 6) {
+      if (day === 5) {
+        setDay(6);
+        return;
+      } else if (day === 12) {
+        setDay(13);
+        return;
+      }
+    }
   };
 
-  const [timeRemaining, setTimeRemaining] = useState(calculateTimeRemaining());
-  const [week, setWeek] = useState('weekOne');
-
-  // This function will display the current activity based on day of the week
+  // This function will find the current activity based on current day and set to state
   const currActivity = () => {
-    const daysOfWeek = [
-      'Sunday',
-      'Monday',
-      'Tuesday',
-      'Wednesday',
-      'Thursday',
-      'Friday',
-      'Saturday',
-    ];
-
     const activityList = {
-      weekOne: {
-        Sunday: 'Voting Session 1',
-        Monday: 'Read',
-        Tuesday: 'Draw',
-        Wednesday: 'Write',
-        Thursday: 'Squadding Up',
-        Friday: 'Point Share',
-        Saturday: 'Matchup',
-      },
-      weekTwo: {
-        Sunday: 'Big Final Reveal',
-        Monday: 'Matchup',
-        Tuesday: 'Voting Session 2',
-        Wednesday: 'Matchup',
-        Thursday: 'Voting Session 3',
-        Friday: 'Matchup',
-        Saturday: 'Big Final Reveal',
-      },
-    };
-
-    let day = daysOfWeek[new Date().getDay()];
-    // let day = daysOfWeek[new Date("2021-08-08T23:59:59").getDay()];
-    // let day = daysOfWeek[new Date("2021-08-09T00:00:00").getDay()];
-
-    const toggleWeek = week => {
-      week === 'weekOne' ? setWeek('weekTwo') : setWeek('weekOne');
+      0: "Big Final Reveal",
+      1: "Read",
+      2: "Draw",
+      3: "Write",
+      4: "Squadding Up",
+      5: "Point Share",
+      6: "Matchup 1",
+      7: "Voting Session 1",
+      8: "Matchup 2",
+      9: "Voting Session 2",
+      10: "Matchup 3",
+      11: "Voting Session 3",
+      12: "Matchup 4",
+      13: "Big Final Reveal"
     };
-
-    if (day === 1 && convertTimeFormat(timeRemaining) === 0) {
-      toggleWeek(week);
-    }
-
-    console.log(
-      'week',
-      week,
-      'day',
-      day,
-      'currActivity',
-      activityList[week][day],
-      'timeRemaining',
-      convertTimeFormat(timeRemaining)
-    );
-    return activityList[week][day];
+    setActivity(activityList[day]);
+    return activity;
   };
 
-  currActivity();
-
+  // useEffect to call both functions on render
   useEffect(() => {
-    const timer =
-      timeRemaining > 0 &&
-      setTimeout(() => setTimeRemaining(calculateTimeRemaining()), 1000);
-    return () => clearTimeout(timer);
-  }, [timeRemaining]);
+    currDay();
+    currActivity();
+  });
```

Comment:
Are there any dependencies? If not, might want to just add an empty dependency array

Reply:
I did have an empty array in here, but eslint kept saying it was not needed.  It is functioning like this, but I can go back and add it on next pass.